src/server_manager/web_app/ui_components/outline-gcp-create-server-app.ts

```diff
@@ -303,32 +307,55 @@ export class GcpCreateServerApp extends LitElement {
     } else {
       if (!this.billingAccounts || this.billingAccounts.length === 0) {
         this.showBillingAccountSetup();
+        // Check every five seconds to see if an account has been added.
+        this.billingAccountsRefreshLoop = window.setInterval(() => {
+          this.refreshBillingAccounts();
+        }, 5000);
       } else {
         this.showProjectSetup(this.project);
       }
     }
   }
 
+  disconnectedCallback() {
+    super.disconnectedCallback();
+    this.stopRefreshingBillingAccounts();
+  }
+
   private init() {
     this.currentPage = '';
     this.selectedProjectId = '';
     this.selectedBillingAccountId = '';
+    this.stopRefreshingBillingAccounts();
   }
 
   private showBillingAccountSetup(): void {
     this.currentPage = 'billingAccountSetup';
   }
 
-  private async handleBillingVerificationNextTap(): Promise<void> {
-    this.showProjectSetup();
-  }
-
-  private async showProjectSetup(existingProject?: Project): Promise<void> {
+  private async refreshBillingAccounts(): Promise<void> {
     this.billingAccounts = await this.account.listBillingAccounts();
```

Comment:
Please catch and log the error. We shouldn't have uncaught exceptions.

Failing silently is ok, since the Next click will check and tell the user the result of the check.

Reply:
OK, I added catch-and-log blocks to both places where this is called.

I don't think silent failures are OK, but we should think more about what the right solution is.  If the call is persistently failing (as happened to me once, maybe due to an expired oauth token), the user will be stuck at this page, without any indication that something is wrong.

Reply:
I agree. I made a suggestion to address that in the new PR.